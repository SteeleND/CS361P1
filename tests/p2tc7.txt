4
0
1 2 3
0a0 0b3 0e1 1b2 2e3 3a4 4b4 4e0
