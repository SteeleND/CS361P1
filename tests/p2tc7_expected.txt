Q = { [1, 0] [2, 3] [1, 4, 0] [] [1, 2, 3, 4, 0] }
Sigma = { a b }
delta =
                   a         b
    [1, 0]    [1, 0]    [2, 3]
    [2, 3] [1, 4, 0]        []
 [1, 4, 0]    [1, 0][1, 2, 3, 4, 0]
        []        []        []
[1, 2, 3, 4, 0] [1, 4, 0][1, 2, 3, 4, 0]
q0 = [1, 0]
F = { [1, 4, 0] [1, 2, 3, 4, 0] }

